pgcopy/util.py

```diff
@@ -41,7 +41,7 @@ def get_schema(conn, table):
         JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
         WHERE c.oid = %s::regclass
         """)
-    cur.execute(query, (psycopg2.sql.Identifier(table),))
+    cur.execute(query, (table,))
```

Comment:
Shouldn’t this also be wrapped?  Here `table` is just the plain string passed to `CopyManager`